### DISCUSSION TITLE
希望降低部署难度

### DISCUSSION THREAD

**Discussion by gospider001** [stoneatom/stonedb#265] [Ideas]
由于系统配置原因不想使用docker部署stonedb，用其他方法总会遇到各种各样的环境和依赖问题。希望可以降低部署难度以使像我这样子的菜鸟用上stonedb

**Comment by RingsC** [comment]
Thanks for your proposal, We are working hard to make stonedb easier to deploy .   You can open an issue to describe the problems you met. or PRs are warmly welcome.
